api/go.mod

```diff
@@ -1,10 +1,32 @@
 module github.com/ceph/ceph-csi/api
 
-go 1.16
+go 1.18
```

Comment:
This requires consumers of the API to use Go 1.18. Ideally we provide the API for the lpwest, still supported Go version?

Reply:
no.., 1.16 and 1.17 are unsupported or already hit EOL.